### PR TITLE
[WIP] Remove symbolic alleles

### PR DESCRIPTION
**Pull request by ernfrid**
This is untested, but I thought I'd put it out here sooner rather than later.

### CODE REVIEW

File: yaps2/pipelines/postvqsr38.py [hunk 1]
```diff
@@ -490,6 +492,37 @@ def create_remove_ac_0_tasks(self):
...
+        lsf_params = get_lsf_params(
+                remove_symbolic_deletion_alleles_lsf_params,
+                self.config.email,
+                self.config.docker
+                )
+        lsf_params_json = to_json(lsf_params)
+
+        for chrom in self.config.chroms:
+            vcf = self.config.vcfs[chrom]
+            output_vcf = 'combined.c{chrom}.vcf.gz'.format(chrom=chrom)
```
Comment (ernfrid) [inline yaps2/pipelines/postvqsr38.py]: @indraniel - Is this the right filename here? It seemed like you re-used this in a couple of places.

**Comment by ernfrid** [comment]
@indraniel - test run on MT appears to have worked...

File: yaps2/pipelines/postvqsr38.py (hunk 1 above)
Comment (indraniel) [reply yaps2/pipelines/postvqsr38.py]: The file name being used doesn't really matter too much in the intermediate steps as long at the files are being written to the correct step directory.  I've been a bit lax about it; just to make the copy and pasting of new steps easier.

**Comment by indraniel** [comment]
Only one very minor nitpick: 

https://github.com/indraniel/yaps2/pull/1/files#diff-eccb7d2097766cbcc265c3bac839c273R510

Could the following output log:

```
output_log = 'remove-symbolic-alleles-chrom-{}-gatk.log'.format(chrom)
```

Have the `-gatk` removed?  The step is using bcftools instead of gatk.  Just not to confuse users who maybe looking at the intermediate steps...

**Comment by indraniel** [comment]
merged in!

**Pull request merged**